extensions/kubernetes/vanilla/deployment/src/main/java/io/quarkus/kubernetes/deployment/KubernetesConfigUtil.java

```diff
@@ -113,6 +115,25 @@ private static void handleExpose(Config config, Map<String, Object> unPrefixed,
         }
     }
 
+    private static void handleExpose(Map<String, Object> unPrefixed, PlatformConfiguration... platformConfigurations) {
+        for (PlatformConfiguration platformConfiguration : platformConfigurations) {
+            if (platformConfiguration.getExposition().isPresent()) {
+                final ExpositionConfig expositionConfig = platformConfiguration.getExposition().get();
+                if (expositionConfig.expose) {
+                    unPrefixed.put(DEKORATE_PREFIX + platformConfiguration.getTargetPlatformName() + "." + EXPOSE_PROPERTY_NAME,
+                            true);
+                    expositionConfig.host.ifPresent(
+                            host -> unPrefixed.put(DEKORATE_PREFIX + platformConfiguration.getTargetPlatformName() + ".host",
+                                    host));
+                    if (expositionConfig.secure) {
```

Comment:
This doesn't have more consequences? Asking that because it's a specific checkbox in the UI.

Reply:
Asking because I remember trying to define the annotation afterwards and things didn't work and George recreated the route entirely and it worked. So I wonder if `secure` is more than just that.

Reply:
Yes that's possible. There is not much about tls in `dekorate` neither, maybe that would be a good use case. WDYT @iocanel?

Reply:
AFAIR, the particular file is meant to be used to provide support for the old style configuration. So, I think that we can entirely ignore that for this pull request. In fact, I think we have it long enough that we should probably get rid of it soon.